jenkins-x/jx
# Issue #3502 Improve pegomock integration

- upgrade to the latest version of pegomock
- each code generation target calls depends now on fmt
- adding verify-generation-complete target
- updating mocks according to latest pegomock version

- stop autogenerating pegomock matchers since they keep changing and easily can be generated manually
- remove generated pegomock matcher structs
- remove heavily mocked tests which are hard to understand and maintain
